app/assets/stylesheets/public/_docs.scss

```diff
@@ -128,8 +128,19 @@ $color-green: rgb(3, 165, 98);
 }
 
 .Docs__attribute__table {
-  colgroup col:nth-child(1) {
-    width: 5rem;
+  table-layout: fixed;
+  width: 100%;
+
+  th {
+    word-wrap: break-word;
+
+    code {
+      white-space: normal;
```

Comment:
Wrap and break words within `th` and `td`. Also wrapping inline `code` so elements like long variables break into a new line and avoids distorting the layout.

Reply:
There were _reasons_ we didn't do this. Mostly around ease of cutting and pasting variable names or keys into scripts and config files. We never satisfactorily resolved it. Perhaps it is time to revisit those ideas, @plaindocs?